manifests/charts/base/values.yaml

```diff
@@ -1,39 +1,40 @@
-global:
+defaults:
```

Comment:
nit: why add a new level `defaults`

Reply:
Its described in the README in the PR and zzz_profile.yaml

```
We have three sets of values, in order of precedence (last wins):
1. The builtin values.yaml defaults
2. The profile the user selects
3. Users input (-f or --set)

Unfortunately, Helm provides us (1) and (3) together (as .Values), making it hard to insert (2).

However, we can workaround this by placing all of (1) under a specific key (.Values.defaults).
We can then merge the profile onto the defaults, then the user settings onto that.
Finally, we can set all of that under .Values so the chart behaves without awareness.
```